java/src/test/java/io/ably/lib/test/common/jre/JreResourceLoader.java

```diff
@@ -10,7 +10,7 @@
 	public byte[] read(String resourceName) throws IOException {
 		FileInputStream fis = null;
 		try {
-			fis = new FileInputStream(new File("src/test/resources", resourceName));
+			fis = new FileInputStream(new File("../lib/src/test/resources", resourceName));
```

Comment:
Is there a requirement to be in a particular directory in order to run the tests?